package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-upendodnn",
-  "version": "1.13.1",
+  "version": "1.13.0",
```

Comment:
@ArielBlanco1990 There is no reason for this version number to be updated in this PR.

Reply:
@WillStrohl This was apparently a mistake I made, I don't remember changing this intentionally. I have reverted this change